docs/tutorials/google/start.ipynb

```diff
@@ -512,6 +512,7 @@
    ],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    import cirq_google as cg
</pre><p>It looks like this is already imported in 2 blocks above. Should we remove the latter of those rather than adding a new one here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/4939/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>This block is actually meant to be run locally rather than inside the notebook</p>

Reply:
:facepalm: I should have looked more thoroughly. Thanks for the clarification!